csrc/ir_nodes.cpp

```diff
@@ -187,11 +190,7 @@ std::string IndexSelectOp::toString(int indent_size) const {
   indent(ss, indent_size) << output(0)->toString() << "\n";
   indent_size++;
   indent(ss, indent_size) << " = index_select( ";
-  if (input(0)->isA<kir::TensorIndex>()) {
```

Comment:
Could you run tests with `$PYTORCH_NVFUSER_DUMP=lower_verbose` to check if this change will be a problem?

Reply:
That's a good point. I blindly replaced it with `getTvInput` and didn't think why. I don't think it's necessary to print the TensorView of the TensorIndex input. That's not done in any other other `toString` methods. Just changed to call `input(0)->toString()`